tests/helpers.test.js

```diff
@@ -70,4 +70,97 @@ describe('helpers', () => {
             expect (clonedObject).toEqual(objectWithArrays);
         });
     });
+
+    describe('isNotEqualObject', () => {
+        it('should return true when objects has diffrent length', () => {
+            const srcObject = { foo: 'bar' };
+            const compareObject = { foo: 'bar', foo2: 'bar2'};
+            expect (isNotEqualObject(srcObject, compareObject)).toBeTruthy();
+        });
+        it('should return true when objects keys has diffrent types', () => {
+            const srcObject = { foo: 'bar' };
+            const compareObject = { foo: true};
+            expect (isNotEqualObject(srcObject, compareObject)).toBeTruthy();
+        });
+        it('should return true when objects keys has diffrent values', () => {
+            const srcObject = { foo: 'bar' };
+            const compareObject = { foo: 'bar2'};
+            expect (isNotEqualObject(srcObject, compareObject)).toBeTruthy();
+        });
+        it('should return true when objects keys are objects and has diffrent values', () => {
+            const srcObject = {
+                foo: {
+                    foo2: 'bar'
+                }
+            };
+            const compareObject = {
+                foo: {
+                    foo2: 'bar2'
+                }
+            };
+            expect (isNotEqualObject(srcObject, compareObject)).toBeTruthy();
+        });
+        it('should return true when objects keys are arrays and has diffrent lengths', () => {
+            const srcObject = {
+                foo: [
+                    'bar',
+                    'bar2'
+                ]
+            };
+            const compareObject = {
+                foo: [
+                    'bar'
+                ]
+            };
+            expect (isNotEqualObject(srcObject, compareObject)).toBeTruthy();
+        });
+        it('should return true when objects keys are arrays and has diffrent values', () => {
+            const srcObject = {
+                foo: [
+                    'bar',
+                    'bar2'
+                ]
+            };
+            const compareObject = {
+                foo: [
+                    'bar',
+                    'bar3'
+                ]
+            };
+            expect (isNotEqualObject(srcObject, compareObject)).toBeTruthy();
+        });
+
+
+        it('should return true when objects keys has no diffrent values', () => {
+            const srcObject = { foo: 'bar' };
+            const compareObject = { foo: 'bar'};
+            expect (isNotEqualObject(srcObject, compareObject)).toBeFalsy();
+        });
+        it('should return true when objects keys are objects and has no diffrent values', () => {
```

Comment:
Is this description ok? I don't see any keys that are objects. I've understood it like so:
```javascript
const foo = {}; // some object
const srcObject = {
  [foo]: {
    foo2: 'bar'
  }
};
```